manage.py

```diff
@@ -26,6 +28,11 @@ def init_db(archive=None):
     More information about populating a PostgreSQL database efficently can be
     found at http://www.postgresql.org/docs/current/static/populate.html.
     """
+    print('Creating user and a database...')
+    exit_code = subprocess.call('sudo -u postgres psql < ' + os.path.join('admin', 'sql', 'create_db.sql'), shell=True)
```

Comment:
If the user already exists and the database doesn't, how will this fail? 

e.g. I just deleted the database before I did a test import, but the user still existed